src/irc/IrcServer.ts

```diff
@@ -37,6 +37,7 @@ export interface IrcServerConfig {
     password?: string;
     allowExpiredCerts?: boolean;
     additionalAddresses?: string[];
+    onlyAdditionalAddresses?: boolean;
```

Comment:
It's optional? Even though we have a default?

Reply:
point